boards/arm/google_twinkie_v2/google_twinkie_v2.dts

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2023 The ChromiumOS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/g0/stm32g0b1Xe.dtsi>
+#include <st/g0/stm32g0b1r(b-c-e)ixn-pinctrl.dtsi>
+
+/ {
+	model = "Google Twinkie V2";
+	compatible = "google,twinkie-v2";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red_led_0: led0 {
+			gpios = <&gpioc 8 GPIO_ACTIVE_LOW>;
+		};
+		green_led_1: led1 {
+			gpios = <&gpiob 6 GPIO_ACTIVE_LOW>;
+		};
+		blue_led_2: led2 {
+			gpios = <&gpiob 7 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		dfu_detect: dfudetect {
+			gpios = <&gpioa 14 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
```

Comment:
This is a bit confusing since there's no actual button, maybe add a note mentioning that this goes to a TP5?